docs/16_dependencies_licenses.md

```diff
@@ -0,0 +1,159 @@
+| lint | [pub.dev](https://pub.dev/packages/lint) | [Apache 2.0](https://github.com/passsy/dart-lint/blob/master/LICENSE) |
+| args | [pub.dev](https://pub.dev/packages/args) | [BSD](https://github.com/dart-lang/args/blob/master/LICENSE) |	
+| mockito | [pub.dev](https://pub.dev/packages/mockito) | [Apache 2.0](https://github.com/dart-lang/mockito/blob/master/LICENSE) |
+| network_image_mock | [pub.dev](https://pub.dev/packages/network_image_mock) | [BSD](https://github.com/stelynx/network_image_mock/blob/master/LICENSE) |
+
+
+## api_mock_server			
+
+| Name | Source | License |
+| :---: | :---: | :---: |
+| http | [pub.dev](https://pub.dev/packages/http) | [BSD](https://github.com/dart-lang/http/blob/master/LICENSE) |
+| dev: |  |  |	
+| lint | [pub.dev](https://pub.dev/packages/lint) | [Apache 2.0](https://github.com/passsy/dart-lint/blob/master/LICENSE) |
+| test | [pub.dev](https://pub.dev/packages/test) | [BSD](https://github.com/dart-lang/test/blob/master/pkgs/test/LICENSE) |
+
+## shell_words		
+
+| Name | Source | License |
+| :---: | :---: | :---: |
+| unicode |[pub.dev](https://pub.dev/packages/unicode) | [BSD](https://github.com/mezoni/unicode/blob/master/LICENSE) |
+| dev: |  |  |	
+| pedantic | [pub.dev](https://pub.dev/packages/pedantic) | [BSD](https://github.com/dart-lang/pedantic/blob/master/LICENSE) |
+| test | [pub.dev](https://pub.dev/packages/test) | [BSD](https://github.com/dart-lang/test/blob/master/pkgs/test/LICENSE) |
+
+## Guardian			
+
+| Name | Source | License |
+| :---: | :---: | :---: |
+| args | [pub.dev](https://pub.dev/packages/args) | [BSD](https://github.com/dart-lang/args/blob/master/LICENSE) |
+| http | [pub.dev](https://pub.dev/packages/http) | [BSD](https://github.com/dart-lang/http/blob/master/LICENSE) |
+| meta | [pub.dev](https://pub.dev/packages/meta) | [BSD](https://github.com/dart-lang/sdk/blob/master/pkg/meta/LICENSE) |
+| xml | [pub.dev](https://pub.dev/packages/xml) | [MIT](https://github.com/renggli/dart-xml/blob/master/LICENSE) |
+| equatable | [pub.dev](https://pub.dev/packages/equatable) | [MIT](https://github.com/felangel/equatable/blob/master/LICENSE) |
+| dev: |  |  |	
+| lint | [pub.dev](https://pub.dev/packages/lint) | [Apache 2.0](https://github.com/passsy/dart-lint/blob/master/LICENSE) |
+| test | [pub.dev](https://pub.dev/packages/test) | [BSD](https://github.com/dart-lang/test/blob/master/pkgs/test/LICENSE) |
+
+## Yaml Map	
+
+| Name | Source | License |
+| :---: | :---: | :---: |
+| yaml | [pub.dev](https://pub.dev/packages/yaml) | [BSD](https://github.com/dart-lang/yaml/blob/master/LICENSE) |
+| dev: |  |  |		
+| lint | [pub.dev](https://pub.dev/packages/lint) | [Apache 2.0](https://github.com/passsy/dart-lint/blob/master/LICENSE) |
+| test | [pub.dev](https://pub.dev/packages/test) | [BSD](https://github.com/dart-lang/test/blob/master/pkgs/test/LICENSE) |
+
+## Third Party API
```

Comment:
Seems like we don't need this section in the document. Instead, as the issue description states, we probably should add information about the actions we use in our Github Actions workflows like `actions/checkout@v2`, `actions/checkout@v2`, `actions/upload-artifact@v2`. 

cc @illia-romanenko

Reply:
As Jenkins and Buildkite CI configuration is in progress, we can skip them for now I think.